recipes/python-semantic-release/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "python-semantic-release" %}
+{% set version = "4.8.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 93933ef00fa23b566f272200c5637a0711742bedd8c63acdeff98d986f624216
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - semantic-release=semantic_release.cli:entry
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click >=2.0,<8.0
+    - gitpython >=2.1,<3
+    - invoke ==0.11.1
```

Comment:
Do you really need this tight of a pin?

Reply:
To be honest, this comes from official repository requirements file: [base](https://github.com/relekang/python-semantic-release/blob/master/requirements/base.txt).

I have submitted a PR in order to bump dependencies (see [173](https://github.com/relekang/python-semantic-release/pull/173)) which includes a range for invoke instead of a fixed pinned version.